content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/spawning.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Spawning Autonomous Player Entities
+description: A reference for spawning and registering network player entities
+linktitle: Spawning Players
+---
+
+When a player joins a multiplayer session, one of the first things that often occurs is spawning a networked entity for them to control. In order to do this, the **Multiplayer** Gem provides an interface to specify an *autonomous* entity for a player. (Refer to the definition of _autonomous role_ in the [Multiplayer Gem Overview](/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/overview#multiplayer-entity-roles).)
```

Comment:
Though this line makes sense, the string of qualifiers in the first sentence might be difficult for some readers to parse. Trying for some clarity.

```suggestion
In most scenarios, when a player joins a multiplayer session, a controllable networked entity must be spawned for the player. The **Multiplayer** Gem provides an interface to specify an *autonomous* entity for a player. Refer to the definition of *autonomous role* in the [Multiplayer Gem Overview](overview#multiplayer-entity-roles) for more information.
```